govulncheck-with-ignore-vulns/govulncheck-with-ignore-vulns.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+user_help () {
+    echo "Runs govulncheck ignoring vulnerabilities from config file"
+    echo "options:"
+    echo "-cfg, --config-file      Path to the configuration file"
+    echo "-h,   --help             To show this help text"
+    echo ""
+    exit 0
+}
+
+read_arguments() {
+    if [[ $# -lt 2 ]]
+    then
+        user_help
+    fi
+
+    while test $# -gt 0; do
+           case "$1" in
+                -h|--help)
+                    user_help
+                    ;;
+                -cfg|--config-file)
+                    shift
+                    CONFIG_FILE=$1
+                    shift
+                    ;;
+                *)
+                   echo "$1 is not a recognized flag!" >> /dev/stderr
+                   user_help
+                   exit -1
+                   ;;
+          esac
+    done
+}
+
+set -e -o pipefail
+
+read_arguments $@
+
+TODAY=$(date -I)
+
+# load vulns to ignore from $CONFIG_FILE using yq
+IGNOREVULNS="$(yq -o=json eval '.ignore' "$CONFIG_FILE")"
+echo IGNOREVULNS: $IGNOREVULNS
```

Comment:
the same for the rest of the code - 
```suggestion
IGNORE_VULNS="$(yq -o=json eval '.ignore' "${CONFIG_FILE}")"
echo IGNORE_VULNS: ${IGNOREVULNS}
```